spec/index.md

```diff
@@ -4065,12 +4084,21 @@ S with
             S.freezing_threshold[A.canister_id],
             S.canister_subnet[A.canister_id].subnet_size,
           );
+          query_stats = noise(SUM {{num_calls_total: 1,
+                                    num_instructions_total: single_query_stats.num_instructions,
+                                    request_payload_bytes_total: single_query_stats.request_payload_bytes,
+                                    response_payload_bytes_total: single_query_stats.response_payload_bytes} |
+                                   single_query_stats <- S.query_stats[A.canister_id];
+                                   single_query_stats.timestamp <= S.time[A.canister_id] - T})
```

Comment:
@Sawchord Could you please confirm that the aggregation takes into account all query stats up until a recent point in time (due to query stats algorithm delay), i.e., that the aggregation does not run over a sliding window?